common/questions.tsx

```diff
@@ -0,0 +1,309 @@
+    content: content,
+    id: String(questionCount), // need to access questionID from questionidtoquestioncontent
+    required: required,
+  };
+}
+
+function makeShortText(
+  content: ReactNode,
+  required: boolean,
+  placeholder?: string
+): ShortText {
+  questionCount++;
+  return {
+    type: QuestionType.ShortText,
+    maxLength: maxLengthShort,
+    minLength: minLengthShort,
+    placeholder,
+    content: content,
+    id: String(questionCount), // need to access questionID from questionidtoquestioncontent
+    required: required,
+  };
+}
+
+function makeDropdown(
+  content: ReactNode,
+  options: Array<string>,
+  required: boolean,
+  placeholder?: string
+): Dropdown {
+  questionCount++;
+  return {
+    type: QuestionType.Dropdown,
+    options: options.map((name) => ({ name })),
+    placeholder,
+    content: content,
+    id: String(questionCount), // need to access questionID from questionidtoquestioncontent
+    required: required,
+  };
+}
+
+function makeLongText(content: ReactNode, required: boolean): LongText {
+  questionCount++;
+  return {
+    type: QuestionType.LongText,
+    maxLength: maxLengthLong,
+    minLength: minLengthLong,
+
+    content: content,
+    id: String(questionCount), // need to access questionID from questionidtoquestioncontent
+    required: required,
+  };
+}
+
+// write questions for portal here
+export const Questions: Array<QuestionDefinition> = [
+  makeShortText('What is your name?', true, 'Nupur Neogi'),
+  makeDropdown(
+    'What is your gender?',
+    [
+      'Male',
+      'Female',
+      'Non-binary',
+      'Genderqueer',
+      'Unlisted',
+      'Prefer not to say',
+    ],
+    true,
+    'Unlisted'
+  ),
+  makeCheckbox(
+    'What ethnicities do you identify as?',
+    [
+      'Indigenous American or Alaska Native',
+      'East Asian',
+      'South/Southeast Asian',
+      'Black/African American',
+      'Hispanic/Latinx',
+      'Native Hawaiian or Other Pacific Islander',
+      'White',
+      'Unlisted',
+      'Prefer not to say',
+    ],
+    true,
+    9
+  ),
+  makeDropdown(
+    'What school do you attend?',
+    [
+      'Northeastern University',
+      'MIT',
+      'Harvard University',
+      'Tufts University',
+      'University of Massachusetts Amherst',
+      'Boston College',
+      'Boston University',
+      'Emerson College',
+      'Suffolk University',
+      'Brandeis University',
+      'Wellesley College',
+      'Wentworth Institute of Technology',
+      'Olin College of Engineering',
+      'Simmons University',
+      'Bristol Community College',
+      'Worcester Polytechnic Institute',
+      'Other',
+    ],
+    true,
+    'Other'
+  ),
+  makeShortText(
+    'If your school was not listed in the previous question, list it here!',
+    false
+  ),
+  makeDropdown(
+    'What level of education are you currently pursuing?',
+    ['High school', 'Undergraduate', 'Graduate'],
+    true,
+    'Undergraduate'
```

Comment:
For the question defaults, can you change it to something that isn't the same as an option? For example, change the default for the "What is your gender" to "Gender". the same for all the other questions